content/blog/2021/2021-12-17-visualizations-101.md

```diff
@@ -0,0 +1,77 @@
+
+{{< window-screenshot src="/blog_img/2021/2021-12-17-distributions.png" >}}
+
+The widget can be used to display differences in distributions.
+
+**How to read the plot**: In the group of patients that are younger than 50 years, there is a 50% lower presence of diameter narrowing (70 % of patients don't have the condition). 
+
+### Sieve Diagram
+
+Sieve diagram is a complex, yet extremely useful visualization. It shows the difference between the expected and observed variable frequency. In other words, if a combination of two values, i.e. being 60 years old and having a heart disease, is more frequent than expected, Sieve diagram will show it.
+
+{{< window-screenshot src="/blog_img/2021/2021-12-17-sieve.png" >}}
+
+In the scatter plot above, we observed a slightly higher incidence of diameter narrowing in patient around 60 years of age. Let us check this is Sieve diagram.
+
+On y axis we see *diameter narrowing*, with the size of each box vertically corresponding to the proportion of instances with and without condition. On x axis we see *age*, with the size of each box horizontally corresponding to the size of each bin. As numeric variables are discretized with equal frequency discretization, the box will always be the same size.
+
+The color of the box corresponds to how overrepresented (blue) or underrepresented (red) a certain combination is. For example, there are 73 patients (24 %) between 55.5 and 60.5 years old in the data set. There are also 139 patients (46%) with diameter narrowing.
+
+If these two variables would be unrelated, we would expect around 11% of patients with both values (0.24 x 0.46 = 0.11), that is patients between 55 and 60 years old with a diameter narrowing.
+
+But if we observe the data set, we can see there are actually 15% of such patients. That is more than we would expect, so a combination of the two values could be statistically significant.
+
+In the left corner of the plot we once again meet the Chi-square, which tell us how significant the combination actually is.
+
+**How to read the plot**: Based on the data set, it is more likely the patients between 55 and 60 years of age will have diameter narrowing. The p-value is below 0.05.
+
+### Word of caution
+
+Data mining is finding patterns in the data. But sometimes, patterns are random and correlation does not necessarily mean causation. So be careful when interpreting the results. Always disclose the size of your data set and how it was gathered. Report the results with the awareness, that you are only observing a sample of the population. And always know that p-values are the chance that the null hypothesis is true. Even if the p-value is lower than 0.05, there's often still non-zero change the null hypothesis is, indeed, true.
```

Comment:
Data mining is about finding patterns
And always know that p-values -> Remember that p-values
change -> chance
oz. cel zadnji stavek bi mogoce lahko zamenjala z
Even if the p-value is 0.05, the null hypothesis could still be true - you would actually expect this to happen 5 times out of a 100, so keep this in mind, especially when you test 100+ variables!